examples/flutter_gallery/lib/demo/cupertino/cupertino_picker_demo.dart

```diff
@@ -70,6 +82,28 @@ class _CupertinoPickerDemoState extends State<CupertinoPickerDemo> {
         );
       }),
     );
+
+    return new GestureDetector(
```

Comment:
can you move things around a bit so utility functions like _buildBottomPicker doesn't appear in between _buildColorPicker and _buildCountdownTimerPicker which are similar?